app/models/calculator_resolver.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class CalculatorResolver
+  def self.call(calculator)
+    dependencies = {}
+    get_fields(calculator).each do |field|
```

Comment:
Consider using each_with_object

Reply:
Thanks. I've used this method. Can you advise what name to choose for hash or array which is used inside `each_with_object` not to repeat the name of the main variable. For example:
`dependencies = get_fields(calculator).each_with_object({}) { |field, hash| }`
Here I used `hash` not to repeat `dependencies`. But it seems that `hash` is too general.